src/libutil/package.nix

```
@@ -17,6 +17,19 @@
 
 let
   inherit (lib) fileset;
+  widecharwidth = stdenv.mkDerivation {
+    name = "widecharwidth";
+    dontUnpack = true;
+    dontStrip = true;
+    installPhase = ''
+      mkdir -p $out/include
+      cp $src $out/include/widechar_width.h
+    '';
+    src = builtins.fetchurl {
```

I would prefer if we clone the whole repository. Also we shouldn't use builtins fetcher.

```suggestion
    src = pkgs.fetchFromGitHub {
```